sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/implementation/models/OcrSkill.java

```diff
@@ -41,30 +34,6 @@
     @JsonProperty(value = "detectOrientation")
     private Boolean shouldDetectOrientation;
 
-    /**
-     * Get the textExtractionAlgorithm property: A value indicating which
-     * algorithm to use for extracting text. Default is printed. Possible
-     * values include: 'Printed', 'Handwritten'.
-     *
-     * @return the textExtractionAlgorithm value.
-     */
-    public TextExtractionAlgorithm getTextExtractionAlgorithm() {
```

Comment:
Would this be an API breaking change for the service? Or was this property only in the preview?

Reply:
Service swagger is in preview version. All changes are going to finalize after this coming release.